arrayadults3/arrayadults3.js

```diff
@@ -0,0 +1,16 @@
+const persons = [
+    { name: 'Rob', age: 25, bsn: 123456, gender: 'male' },
+    { name: 'Arla', age: 15, bsn: 257654, gender: 'female' },
+    { name: 'Pam', age: 21, bsn: 315421, gender: 'female' },
+    { name: 'Beer', age: 46, bsn: 463456, gender: 'x' },
+    { name: 'Pippi', age: 13, bsn: 533446, gender: 'female' }
+];
+
+function getAdults() {
+    let adults = persons.filter(person => person.age >= 18).map(person => person.name); 
+    console.log(`${adults.join( `, `)} are adults.`)
```

Comment:
Maybe good to practice the `"get"` functions like `getAdults()` should return a value. In this case the `adults` variable.

And move the console.log statement to line 15, after calling `getAdults()`